src/model.coffee

```diff
@@ -270,7 +270,10 @@ class Model extends Backbone.Model
       when 'create'
         blacklist = blacklist.concat @createJSONBlacklist()
       when 'update'
-        blacklist = blacklist.concat @updateJSONBlacklist()
+        if this.useUpdateWhitelist && this.get('update_whitelist')
```

Comment:
We should mirror how we do the blacklist and rather than use a property, we should define a method (`updateJSONWhiteList()`) which we default to an empty function and return an array of attributes that we'd want to use as a whitelist.